### DISCUSSION TITLE
Bus Factor Renaming Poll

### DISCUSSION THREAD

**Discussion by geekygirldawn** [chaoss/community#634] [Polls]
Before voting, please read the full discussion in the [Bus Factor Renaming](https://github.com/chaoss/community/issues/632) issue to understand the context and why we are considering renaming this metric.

**DO NOT** leave comments here on the poll. If you have additional comments, please add them to the issue linked above to avoid fragmenting the discussion into 2 places. 

Please keep in mind that this is a poll to **gauge reactions** with the caveat that the CHAOSS project operates by consensus, so this poll is limited to understanding what people think, and the one with the most votes will not necessarily be the "winner".

Additional Notes:

- I'm using contributor / person rather than maintainer / developer because this is a decision we've made when defining other metrics to make sure that we don't discount the work of other types of contributors.
- I've left out ones that created confusion and required extra explanation (e.g., majority, pony)
- Anything in [Brackets] contains similar words that we can decide on later after more research.

This poll will remain open for about a month to give all WGs and the broader community further options to discuss this. The poll will close on May 4.

**Comment by decause-gov** [comment]
After visiting OSSNA this year, "Nebraska Factor" a la XKCD (https://xkcd.com/2347/) is a name that has been bigtime popularized in the Supply Chain world. Perhaps this could be a decent candidate, if the namespace is not already claimed elsewhere in CHAOSS?

**Reply by germonprez** [reply]
As a Nebraskan, I can get behind this!

**Comment by geekygirldawn** [comment]
The poll was scheduled to remain open until May 4, so closing this now.
